src/exceptions.cc

```diff
@@ -76,6 +76,7 @@ const std::unordered_map<unsigned, valhalla::valhalla_exception_t> error_codes{
     {163, {163, "Invalid date_type", 400, HTTP_400, OSRM_INVALID_VALUE, "wrong_date_type"}},
     {164, {164, "Invalid shape format", 400, HTTP_400, OSRM_INVALID_VALUE, "wrong_shape_format"}},
     {165, {165, "Date and time required for destination for date_type of invariant", 400, HTTP_400, OSRM_INVALID_OPTIONS, "missing_invariant_date"}},
+    {166, {166, "Exceeded max distance", 400, HTTP_400, OSRM_INVALID_VALUE, "too_large_distance"}},
```

Comment:
in `loki/isochrone_action.cc` we already throw with this error code, but it didn't exist: 

https://github.com/valhalla/valhalla/blob/8f652c83f9cab912b8925ead06befe37acb55ad2/src/loki/isochrone_action.cc#L48-L54